.codex/hooks/pre_tool_use_policy_windows.ps1

```
@@ -0,0 +1,68 @@
+[CmdletBinding()]
+param()
+
+Set-StrictMode -Version Latest
+$ErrorActionPreference = "Stop"
+
+function Fail-Launcher {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Fail-Launcher' uses an unapproved verb.

(PSUseApprovedVerbs)